roofit/hs3/src/RooJSONFactoryWSTool.cxx

```diff
@@ -1598,6 +1609,45 @@ bool RooJSONFactoryWSTool::importYML(std::string const &filename)
    return this->importYML(infile);
 }
 
+void RooJSONFactoryWSTool::importVarfromString(const std::string &jsonString)
+{
+   std::unique_ptr<JSONTree> tree = JSONTree::create(jsonString);
+   const JSONNode &n = tree->rootnode();
+
+   _domains = std::make_unique<Domains>();
```

Comment:
```suggestion
   _domains = std::make_unique<RooFit::JSONIO::Detail::Domains>();
```
Sorry, with the current ROOT master it doesn't compile anymore like this :( You need to qualify the full class name.